Implementation/FindMyBLEDevice/FindMyBLEDevice/App.xaml.cs

```diff
@@ -68,32 +68,5 @@ public App()
 
             MainPage = new AppShell();
         }
-
-        protected override void OnStart()
```

Comment:
Maybe we could leave `OnStart`, `OnSleep` and `OnResume`, since they could be useful in the future.
But let's just keep that in mind, if we need the functionality in the future, we can add them again anytime.